docs/LogGuide.md

```diff
@@ -1122,6 +1139,25 @@ You cannot count on the time remaining to be precise.
 In rare cases, the time will already have counted down a tiny bit.
 This matters for cases such as ucob Nael phase doom debuffs.
 
+The 'stacks' value may also indicate other information about the buff. For example, Mudra will show different stack
+values for different combinations of Mudra. The only way to be sure that you are getting a "real" stack value is by
+cross-referencing with game data (such as via Xivapi) to see what the game says the max stack amount for that ability
+is (0 is the max for most abilities).
+
+#### Refreshes, Overwrites, and Deaths
+
+If a buff is refreshed early, you will get another 26-line. You will not get a 30-line indicating that the existing
+buff has been removed. The same is true for when stacks are consumed or added.
```

Comment:
This is not always true.  I have seen plenty of examples where you lose 4 stacks and gain 5 stacks when transitioning from 4 to 5.  I can dig some up if you need them.  I think light rampant lightsteeped stacks do this, for example.

Reply:
It's almost never true, just a consequence of how ACT's status manager logs events.